SwedbankPaySDK/Classes/SwedbankPaySDKController.swift

```diff
@@ -110,37 +122,36 @@ public final class SwedbankPaySDKController: UIViewController {
     }
     
     /// Initializes the Swedbank Pay SDK, and depending on the `consumerData`, starts the payment process with consumer identification or anonymous process
-    /// - parameter configuration: Configuration object containing `backendUrl`, `headers`, `domainWhitelist` and `pinPublicKeys`; of these, `domainWhitelist` and `pinPublicKeys` are *optional*
+    /// - parameter configuration: Configuration object that handles creating and manipulating Consumer Identification Sessions and Payment Orders as needed.
     /// - parameter consumer: consumer identification information; *optional* - if not provided, consumer will be anonymous
     /// - parameter paymentOrder: the payment order to create
-    public init(configuration: SwedbankPaySDK.Configuration, consumer: SwedbankPaySDK.Consumer? = nil, paymentOrder: SwedbankPaySDK.PaymentOrder) {
+    public convenience init(configuration: SwedbankPaySDKConfiguration, consumer: SwedbankPaySDK.Consumer? = nil, paymentOrder: SwedbankPaySDK.PaymentOrder) {
+        self.init(configuration: configuration, withCheckin: consumer != nil, consumer: consumer, paymentOrder: paymentOrder, userData: nil)
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 140 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)